codespell_lib/data/dictionary.txt

```diff
@@ -9357,6 +9357,7 @@ decrasing->decreasing, deceasing,
 decration->decoration
 decreace->decrease
 decreas->decrease
+decreate->decrease, discrete,
```

Comment:
Maybe also the literal de-create
```suggestion
decreate->decrease, discrete, destroy,
```
Or the bad thing, but perhaps too unlikely, but maybe not much more than discrete:
```suggestion
decreate->decrease, discrete, desecrate,
```

Reply:
> Or the bad thing, but perhaps too unlikely, but maybe not much more than discrete:

Did you want to add desecrate too? You'd need to do some manual editing! :laughing:

Reply:
Are you also suggesting `de-create` as a typo or solution?

Reply:
@peternewman :point_up_2:

Reply:
bump

Reply:
@peternewman ?

Reply:
As a solution.